### ISSUE TITLE
Logstash Repo not really supported

### ISSUE THREAD

**Issue by davedash** [brandentimm/logstash_forwarder-formula#3]
This repo:

https://github.com/brandentimm/logstash_forwarder-formula/blob/master/logstash_forwarder/repo.sls#L10

Is not supported, as noted by: https://github.com/elasticsearch/logstash-forwarder/issues/184#issuecomment-73417804

There should be a more recommended way of obtaining this package.

**Comment by brandentimm** [comment]
Thanks for the report, I hadn't noticed it.  I'll look into building from git's master for the project when I have time, otherwise pull requests are welcome as always!

**Comment by brandentimm** [comment]
Well, from looking at their build instructions, this will be annoying.

Building the software requires golang, which I'm not opposed to, but since so many people have carefully managed Go installations, I'm loathe to install it just to build the software and possibly add a conflicting Go runtime to somebody's system.

It's even worse if we wanted to package it on-system, as in addition to golang we'd need ruby, gem, and a specific gem 'fpm'.  Again, many people have carefully managed ruby/gem setups, that I would be loathe to trample

This formula may just need to sit idle until the logstash-forwarder folks figure out a reasonable packaging strategy.

**Comment by davedash** [comment]
Yeah - or let people supply their own repo that has the package as a pillar?

I'm signing up for packagelcoud.io just so I can build this package locally
and publish it there - kind of involved for just one package, so it might
take me a while, if I have something decent I can submit a PR.

**Comment by brandentimm** [comment]
I like your idea of leaving the package repository url as a bit of pillar info.  We'd have to remove the GPG key step too, and update the README, but those are pretty trivial changes.

I'll see what I can work up this evening.  Thanks for the second set of eyes.
